docs/upgrade.html

```diff
@@ -21,6 +21,9 @@
 
 <h4><a id="upgrade_2_2_0" href="#upgrade_2_2_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x, 1.1.x, 2.0.x or 2.1.x to 2.2.0</a></h4>
 
+<p><b>Note that 2.2.x contains a change to the internal schema used to store consumer offsets. Once the upgrade is
```

Comment:
This is not quite right. The change was in 2.1.x. I would suggest this:
> If you are upgrading from a version prior to 2.1.x, please see the note below about the change to the schema used to store consumer offsets. Once you have changed the inter.broker.protocol.version to the latest version, it will not be possible to downgrade to a version prior to 2.1.